contracts/governance/utils/Votes.sol

```diff
@@ -152,21 +163,37 @@ abstract contract Votes is IVotes, Context, EIP712 {
     function _moveDelegateVotes(address from, address to, uint256 amount) private {
         if (from != to && amount > 0) {
             if (from != address(0)) {
-                (uint256 oldValue, uint256 newValue) = _delegateCheckpoints[from].push(_subtract, amount);
+                (uint256 oldValue, uint256 newValue) = _push(
+                    _delegateCheckpoints[from],
+                    _subtract,
+                    SafeCast.toUint224(amount)
```

Comment:
Since we defined the `_push` function, can we move the cast inside of it?

Reply:
I thought `_push` would be cleaner using uint224 for both the operator function AND the delta.

Reply:
I see what you mean. I agree.